pkg/expr/query.go

```diff
@@ -77,6 +77,7 @@ type ClassicQuery struct {
 // SQLQuery requires the sqlExpression feature flag
 type SQLExpression struct {
 	Expression string `json:"expression" jsonschema:"minLength=1,example=SELECT * FROM A LIMIT 1"`
+	Format     string `json:"format"`
```

Comment:
Might be worth documenting here what are the currently expected format strings? My first assumption in reading this was that it would be long vs wide and then realized it's alerting vs undefined